platform.md

```diff
@@ -566,6 +567,10 @@ Usage:
       - All run-image layers SHALL be preserved
       - All run-image config values SHALL be preserved unless this conflicts with another requirement
     - MUST contain all buildpack-provided launch layers as determined by the [Buildpack Interface Specfication](buildpack.md)
+    - MUST contain a layer containing all buildpack-provided standardized Bill of Materials (sBOM) files for `launch` as determined by the [Buildpack Interface Specfication](buildpack.md)
```

Comment:
Shouldn't this be a **MAY** given that they are optional?

```suggestion
    - MAY contain a layer containing all buildpack-provided standardized Bill of Materials (sBOM) files for `launch` as determined by the [Buildpack Interface Specfication](buildpack.md)
```

Reply:
It is optional for the buildpack to provide them, but if buildpacks have provided an sBOM, then the platform MUST include it (that was my reading).

Reply:
Clarified in cf67713